### PR TITLE
Implement "Get the User's Queue"

### PR DESCRIPTION
**Pull request by ljufa**
## Description

Missing endpoint `/v1/me/player/queue` implemented
https://developer.spotify.com/documentation/web-api/reference/#/operations/get-queue

<br class="Apple-interchange-newline">

## Motivation and Context

It was missing
## Dependencies 
No
## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## How has this been tested?

Model test

Please also list any relevant details for your test configuration

- [ ] Test A: xxx
- [ ] Test B: xxx

## Is this change properly documented?

Please make sure you've properly documented the changes you're making.

Don't forget to add an entry to the CHANGELOG if necessary (new features, breaking changes, relevant internal improvements).

### CODE REVIEW

File: tests/test_models.rs [hunk 1]
```diff
@@ -854,6 +854,267 @@ fn test_current_playback_context() {
     assert!(current_playback_context.progress.is_none());
 }
 
+#[test]
```
Comment (ramsayleung) [inline tests/test_models.rs:857]: Thanks for your contribution, is it possible to minimize the Json data, for example, reduce multiple objects in an  array to a single object?

**Comment by ramsayleung** [comment]
There is one more thing, I think it's necessary to add an entry to the `CHANGELOG.md`, this PR adds a new endpoint.

**Comment by ljufa** [comment]
thank you for quick review, both changes addressed.

**Comment by ramsayleung** [comment]
The CI checks fail, you need to fix the Clippy and Format error before merging this PR :)

**Review by ramsayleung — APPROVED** [review]

**Comment by ramsayleung** [comment]
Merged, thanks again for your contribution :)

**Pull request merged**